### PR TITLE
SOAR-3688: fix spelling 

### PR DESCRIPTION
**Pull request by cstuart-r7**
### Description

Describe the proposed changes:

  - This changes corrects misspelled words in the help.md files of a few plugins. Bumps the patch version and regenerated each affected plugin.

### Testing

Any testing information goes here.

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [x] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [x] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [x] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [x] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [x] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [x] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [x] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [x] Work fully completed
- [x] Functional
  - [x] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [x] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [x] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [x] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [x] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: carbon_black_response/help.md [hunk 1]
```diff
@@ -20,11 +20,16 @@ This plugin utilizes the [VMware Carbon Black EDR REST API](https://developer.ca
...
-|url|string|https://127.0.0.1/api/bit9platform/v1|True|Carbon Black Server API URL|None|
-|ssl_verify|boolean|True|True|SSL certificate verification|None|
-|api_key|credential_secret_key|None|True|API token found in your Carbon Black profile|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|api_key|credential_secret_key|None|True|API token found in your Carbon Black profile|None|None|
+|ssl_verify|boolean|True|True|SSL certificate verification|None|None|
+|url|string|https://127.0.0.1/api/bit9platform/v1|True|Carbon Black Server API URL|None|None|
+
+Example input:
```
Comment (jmcadams-r7) [inline carbon_black_response/help.md:29]: Find and replace and get rid of the empty example fields.

File: elasticsearch/help.md [hunk 2]
```diff
@@ -97,16 +97,6 @@ This action is used to search for documents.
 Example input:
 
 ```
```
Comment (jmcadams-r7) [inline elasticsearch/help.md:99]: Need to readd this example.

File: microsoft_teams/help.md [hunk 3]
```diff
@@ -195,7 +192,7 @@ Example input:
 ```
 {
   "channel_name": "InsightConnect Channel",
-  "message_content": "<b>Hello!</b>",
+  "message_content": "\u003cb\u003eHello!\u003c/b\u003e",
```
Comment (jmcadams-r7) [inline microsoft_teams/help.md:195]: Need to revert this.

File: okta/help.md [hunk 4]
```diff
@@ -121,9 +121,6 @@ This action returns an object containing all of a user's factors for MFA.
 Example input:
 
 ```
-{
-  "user_id" : "opfpfac5jbFkZppdt0h7"
-}
 ```
```
Comment (jmcadams-r7) [inline okta/help.md:124]: Need to revert and re-add all the examples.

File: sentinelone/help.md [hunk 5]
```diff
@@ -32,10 +32,7 @@ Example input:
 
 ```
 {
-  "credentials": {
-    "username": "user@example.com",
-    "password": "mypassword"
-  },
+  "credentials": "{\"username\": \"user@example.com\", \"password\": \"mypassword\"}",
```
Comment (jmcadams-r7) [inline sentinelone/help.md:35]: Revert

File: sentinelone/help.md [hunk 6]
```diff
@@ -60,11 +57,7 @@ Example input:
 ```
 {
   "action": "connect",
-  "filter": {
-    "ids": [
-      "1000000000000000000"
-    ]
-  }
+  "filter": "{\"ids\": [\"1000000000000000000\"]}"
```
Comment (jmcadams-r7) [inline sentinelone/help.md:60]: Revert

File: sonicwall_capture_client/help.md [hunk 7]
```diff
@@ -24,10 +24,7 @@ Example input:
 
 ```
 {
-  "credentials": {
-    "username": "user@example.com",
-    "password": "mypassword"
-  }
+  "credentials": "{\"username\": \"admin\", \"password\": \"mypassword\"}"
```
Comment (jmcadams-r7) [inline sonicwall_capture_client/help.md:27]: Revert

File: urlscan/help.md [hunk 8]
```diff
@@ -28,9 +28,7 @@ Example input:
 
 ```
 {
-  "api_key": {
-    "secretKey": “381cd93b-1946-9c73-1946-c916075eb9a3"
-  }
+  "api_key": "{\"secretKey\": \"381cd93b-1946-9c73-1946-c916075eb9a3\"}"
```
Comment (jmcadams-r7) [inline urlscan/help.md:31]: Revert

**Review by jmcadams-r7 — CHANGES_REQUESTED** [review]

**Review by jmcadams-r7 — APPROVED** [review]

**Review by ncramer-r7 — APPROVED** [review]

**Pull request merged**